site/docs/ch/meta-service/table-properties.md

```diff
@@ -68,6 +68,15 @@
 | log-store.data-version             | v1               | Logstore 中消息的版本，当前仅支持 v1           |
 | log.consistency.guarantee.enable   | false            | 标记是否开启一致性保证                       |
 
+### Watermark 相关配置
+
+| 配置名称                                       | 默认值               | 描述                                     |
+| ----------------------------------------------| ------------------- | ----------------------------------       |
+| table.event-time-field                        | _ingest_time        | 计算 watermark 的事件时间字段，默认的 _ingest_time 表示使用数据写入时间来计算 |
```

Comment:
Should this field(`_ingest_time`) be exists in the table schema?

Reply:
Not needed，`_ingest_time` is a a special configuration like the `process time` in Flink Job.